<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="windowModality">
   <enum>Qt::WindowModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>549</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Расчет на прочность</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>800</width>
      <height>509</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="acceptDrops">
     <bool>false</bool>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="tabShape">
     <enum>QTabWidget::Triangular</enum>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <property name="usesScrollButtons">
     <bool>true</bool>
    </property>
    <property name="documentMode">
     <bool>false</bool>
    </property>
    <widget class="QWidget" name="tab_3">
     <attribute name="title">
      <string>Страница</string>
     </attribute>
     <widget class="QPushButton" name="pb_ob">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>60</y>
        <width>145</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Цилиндрическая обечайка</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pb_kon">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>90</y>
        <width>114</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Конический переход</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pb_el">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>120</y>
        <width>123</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Эллиптическое днище</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_4">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>150</y>
        <width>136</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Полусферическое днище</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_5">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>180</y>
        <width>135</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Торосферическое днище</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_6">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>210</y>
        <width>149</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Пологое коническое днище</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_7">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>240</y>
        <width>144</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Крутое коническое днище</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_8">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>270</y>
        <width>87</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Плоское днище</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_9">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>300</y>
        <width>140</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Плоское днище с ребрами</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_10">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>330</y>
        <width>101</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Овальная крышка</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_11">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>360</y>
        <width>206</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Сферическое неотбортованное днище</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_12">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>60</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Штуцер</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_13">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>90</y>
        <width>101</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Овальный штуцер</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_14">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>390</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Отвод</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_15">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>420</y>
        <width>127</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Фланцевое соединение</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_16">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>160</y>
        <width>101</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Отьемная крышка</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pbSaddle">
      <property name="geometry">
       <rect>
        <x>410</x>
        <y>60</y>
        <width>91</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Седловая опора</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_word">
     <attribute name="title">
      <string>Вывод в Word</string>
     </attribute>
     <widget class="QLabel" name="label_17">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>30</width>
        <height>13</height>
       </rect>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Image&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QListView" name="lvCalc">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>220</y>
        <width>300</width>
        <height>171</height>
       </rect>
      </property>
      <property name="contextMenuPolicy">
       <enum>Qt::CustomContextMenu</enum>
      </property>
      <property name="editTriggers">
       <set>QAbstractItemView::DoubleClicked|QAbstractItemView::EditKeyPressed|QAbstractItemView::SelectedClicked</set>
      </property>
      <property name="movement">
       <enum>QListView::Static</enum>
      </property>
      <property name="modelColumn">
       <number>0</number>
      </property>
     </widget>
     <widget class="QPushButton" name="pbMakeWord">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>460</y>
        <width>101</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Оформить расчет</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_sprav">
    <property name="title">
     <string>Справка</string>
    </property>
    <addaction name="action_about"/>
    <addaction name="separator"/>
    <addaction name="action_close"/>
   </widget>
   <widget class="QMenu" name="menu_file">
    <property name="title">
     <string>Файл</string>
    </property>
    <addaction name="action_open"/>
    <addaction name="action_save"/>
   </widget>
   <addaction name="menu_file"/>
   <addaction name="menu_sprav"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="action_about">
   <property name="text">
    <string>О программе</string>
   </property>
  </action>
  <action name="action_close">
   <property name="text">
    <string>Выход</string>
   </property>
  </action>
  <action name="action_open">
   <property name="text">
    <string>Открыть расчет</string>
   </property>
  </action>
  <action name="action_save">
   <property name="text">
    <string>Сохранить расчет</string>
   </property>
  </action>
 </widget>
 <tabstops>
  <tabstop>tabWidget</tabstop>
  <tabstop>lvCalc</tabstop>
  <tabstop>pbMakeWord</tabstop>
 </tabstops>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections/>
</ui>
